schemas/test/en/test_theme_health.json

```
@@ -0,0 +1,75 @@
+{
+    "mime_type": "application/json/ons/eq",
+    "language": "en",
+    "schema_version": "0.0.1",
+    "data_version": "0.0.3",
+    "survey_id": "0",
+    "title": "Test Health Survey",
+    "theme": "health",
+    "description": "A questionnaire to demo the health survey theme",
+    "metadata": [
+        {
+            "name": "user_id",
+            "type": "string"
+        },
+        {
+            "name": "period_id",
+            "type": "string"
+        },
+        {
+            "name": "ru_name",
+            "type": "string"
+        }
```

Minor but we should prob use social claims for this schema, like `test_theme_social`  (i know questionnaire_id will be renamed soon) e.g. 
```suggestion
        {
            "name": "case_ref",
            "type": "string"
        },
        {
            "name": "questionnaire_id",
            "type": "string"
        },
```